src/CommunityToolkit/Xamarin.CommunityToolkit/Views/Popup/BasePopup.shared.cs

```diff
@@ -18,7 +18,6 @@ public abstract class BasePopup : VisualElement
 		/// </summary>
 		protected BasePopup()
 		{
-			Color = Color.White;
```

Comment:
@pictos do you think we may remove this line. It will be a breaking change, I guess..

Reply:
We have to remove this or change it to Color = Color.Default in order to make the dark mode work on iOS

Reply:
PopUp control isn't in the stable package, so it's fine to break it.

Reply:
@geskill can you change it to be `Color = Color.Default`? Just to make sure? I'm not an expert in the PopUp code base and maybe it can be set with a different value somewhere in some scenario.

Reply:
@pictos as I can see ColorProperty has "Color.Default" as its default value.
So, it should work even w/out this change.